### DISCUSSION TITLE
Windows Uninstall

### DISCUSSION THREAD

**Discussion by 090media** [webcamoid/webcamoid#541] [Ideas]
It would be nifty if the installed application appeared in the Windows "Programs and Features" list for easy uninstall instead of hunting for it in the Start Menu. Thx!

**Comment by hipersayanX** [comment]
The uninstaller is supposed to do that already, Webcamoid 9 use [NSIS](https://nsis.sourceforge.io/Download) for installing the program :confused:

**Comment by 090media** [comment]
I see the uninstaller in the start menu, but the program is not in the "Programs and Features" list, which is how most people manage installed applications in Windows.

**Comment by hipersayanX** [comment]
I'll move this to an issue, it sound more like bug, because the installer is supposed to do that already.
